src/legacy/ui/public/vis/editors/default/agg_params.html

```diff
@@ -13,12 +13,6 @@
     style="display: none;">
 </div>
 
-<div ng-if="agg.error" class="form-group">
-  <p class="visEditorAggParam__error ng-binding">
-    {{agg.error}}
```

Comment:
Now this error message is displayed in aggregation selector control: https://github.com/elastic/kibana/pull/37979/files?file-filters%5B%5D=.tsx&owned-by%5B%5D=#diff-9c2f5870ef0ffd3edbde4d2072e86430R98

Reply:
Can you also change the exclamation point of the error message to just a period:

<img src="https://d.pr/free/i/xjHl8Y+" width="50%" />

Reply:
Done.